six/chain.json

```diff
@@ -0,0 +1,98 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "six",
+  "status": "live",
+  "network_type": "mainnet",
+  "pretty_name": "SIX Protocol",
+  "chain_id": "sixnet",
+  "bech32_prefix": "6x",
```

Comment:
Please register this prefix
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

Reply:
https://github.com/satoshilabs/slips/pull/1637